### ISSUE TITLE
Icon Misconfiguration at Font Sizes Below Default 12pt (phpStorm)

### ISSUE THREAD

**Issue by hventura** [krasa/IdeaIconPack#13]
Following the recent update to the latest version of PHPStorm, I encountered an issue where all interface icons appeared enlarged beyond their standard size.

This issue manifested after modifying the font size to 10pt from the default setting of 12pt (navigated through Settings -> Appearance & Behavior -> Appearance -> Use custom font: Inter; Size set to 10pt). Upon reverting the font size back to the default 12pt and applying the changes, the icon sizes returned to their normal appearance.

I am not sure if the same issue occurs in other JetBrains products!

![phpStormBug](https://github.com/krasa/IdeaIconPack/assets/27702303/5631eae1-2ef7-46a3-a0b4-f17311bca691)

**Comment by krasa** [comment]
Nothing I can do about that. This is all about icon scaling by the IDE. You could report it to them.

**Comment by krasa** [comment]
It may be fixed already https://youtrack.jetbrains.com/issue/IDEA-339663/Icons-and-some-UI-are-scaled-too-much

**Comment by hventura** [comment]
Yes, sorry @krasa. After reading for a while about it i found that it happens a lot after some updates. Can't know for sure what triggers it, but yes, nothing wrong with the plugin :) Love it by the way. 
Sorry for open this issue, can i close it with a comment, or you close it?
